solutions/devsprint-riccardo-1/OnboardingChallengeKit/Sources/OnboardingChallengeKit/Screens/List/ListViewController.swift

```diff
@@ -65,6 +66,12 @@ public final class ListViewController: UIViewController {
             self.listView.updateView(with: configuration)
         }
     }
+    
+    private func instanceDetails(of item: RepositoryCellViewConfiguration) {
```

Comment:
Talking about navigation, here we are coupling the `DetailViewController` with the `ListViewController`. If tomorrow we want to use a different class for the details we would have to update the list view controller as well. Can you think about an alternative way to decouple the navigation from the `ListViewController` to the `DetailViewController`?

*hint:* you already did something similar in the `tableView(_:didSelectRowAt:)`.